README.md

```diff
@@ -1 +1,8 @@
-# Ksoc Terraform Provider (terraform-provider-ksoc)
+# KSOC Terraform Provider
```

Comment:
I highly recommend to install `markdownlint` -  as your IDE extension. Probably it will find few minor things to fix